searx/plugins/limiter.py

```diff
@@ -40,6 +42,39 @@
     + r')'
 )
 
+PING_KEY = 'SearXNG_limiter.ping'
+TOKEN_KEY = 'SearXNG_limiter.token'
+
+
+def ping():
+    redis_client = redisdb.client()
+    user_agent = request.headers.get('User-Agent', 'unknown')
+    x_forwarded_for = request.headers.get('X-Forwarded-For', '')
+
+    ping_key = PING_KEY + user_agent + x_forwarded_for
+    redis_client.set(secret_hash(ping_key), 1, ex=600)
```

Comment:
>> But the URL of the limiter is changed every 10 min by a random token / compare my comments above.

> It doesn't really matter, if the user does multiple searches in a row then for the second time he visits the search page the "ping" CSS resource will be cached, and the issue may appear.

The token is also stored for 600 sec.  Only when the token changes after 10min, it must be "unlocked" again.